aws/resource_aws_macie2_member_test.go

```diff
@@ -0,0 +1,239 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/macie2"
+	"github.com/hashicorp/aws-sdk-go-base/tfawserr"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/terraform-providers/terraform-provider-aws/aws/internal/envvar"
+)
+
+func testAccAwsMacie2Member_basic(t *testing.T) {
+	var providers []*schema.Provider
+	var macie2Output macie2.GetMemberOutput
+	resourceName := "aws_macie2_member.test"
+	email := "required@example.com"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			testAccAlternateAccountPreCheck(t)
+		},
+		ProviderFactories: testAccProviderFactoriesAlternate(&providers),
+		CheckDestroy:      testAccCheckAwsMacie2MemberDestroy,
+		ErrorCheck:        testAccErrorCheck(t, macie2.EndpointsID),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAwsMacieMemberConfigBasic(email),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsMacie2MemberExists(resourceName, &macie2Output),
+					resource.TestCheckResourceAttr(resourceName, "relationship_status", macie2.RelationshipStatusCreated),
```

Comment:
The `_basic` test should also check the values returned by

* `status`
* `account_id`
* `administrator_account_id`
* `master_account_id`

The `...account_id` values can be checked using `resource.TestCheckResourceAttrPair`

Reply:
We should add a check for `status` here